build_test_all.cmd

```diff
@@ -19,6 +19,7 @@ if "%target_platform%"=="arm64" goto :eof
 call msbuild /m /p:Configuration=%target_configuration%,Platform=%target_platform%,CppWinRTBuildVersion=%target_version% cppwinrt.sln /t:cppwinrt
 _build\%target_platform%\%target_configuration%\cppwinrt.exe -in local -out _build\%target_platform%\%target_configuration% -verbose
 call msbuild /p:Configuration=%target_configuration%,Platform=%target_platform%,CppWinRTBuildVersion=%target_version% natvis\cppwinrtvisualizer.sln
+call msbuild /p:Configuration=%target_configuration%,Platform=%target_platform%,CppWinRTBuildVersion=%target_version% test\nuget\NugetTest.sln
```

Comment:
I think you still need a nuget restore for NugetTest.sln - try running "git clean fdx" and running "build_test_all" - that should work.

Reply:
NuGetTest.sln doesn't have any NuGet package dependencies so shouldn't be necessary.

Reply:
Oh it grabs cppwinrt locally? Cool.

Reply:
Turns out I was wrong. The C# project has nuget dependencies.